rtil/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["oberien <jaro.fietz@gmx.de>"]
 edition = "2021"
 
 [dependencies]
+opener = "0.6.1"
```

Comment:
Is there a reason you preferred `opener` over `open`?

Reply:
I couldn't get `open` to be recognized in the project, even after adding it to the cargo.toml file and running `cargo build`, `cargo update` and `make`, nor in a test project.